main.c

```diff
@@ -813,8 +813,8 @@ int __attribute__((used)) main(void) {
     // no SPI flash filesystem, and we might erase the existing one.
 
     // Check whether CIRCUITPY is available. Don't check if it already hasn't been found.
```

Comment:
I thought I needed to reset into safe mode to get the message to print, but I see that is not the case. So the comment is now a bit misleading.
```suggestion
    // Check whether CIRCUITPY is available.
```

Reply:
The broadcom port can't reset (but may have filesystem issues) so I had a reason to switch this.